packages/backend/src/open_payments/payment/outgoing/service.test.ts

```diff
@@ -491,6 +494,60 @@ describe('OutgoingPaymentService', (): void => {
       })
     })
 
+    test(
+      'create many outgoing payments against one grant',
+      withConfigOverride(
+        () => config,
+        { slippage: 0 },
+        async (): Promise<void> => {
+          const debitAmount = {
+            value: BigInt(10),
+            assetCode: receiverWalletAddress.asset.code,
+            assetScale: receiverWalletAddress.asset.scale
+          }
+          const grant: Grant = {
+            id: uuid(),
+            limits: {
+              debitAmount: {
+                value: BigInt(Number.MAX_SAFE_INTEGER),
+                assetCode: receiverWalletAddress.asset.code,
+                assetScale: receiverWalletAddress.asset.scale
+              },
+              receiveAmount: {
+                value: BigInt(Number.MAX_SAFE_INTEGER),
+                assetCode: receiverWalletAddress.asset.code,
+                assetScale: receiverWalletAddress.asset.scale
+              }
+            }
+          }
+          await OutgoingPaymentGrant.query(knex).insertAndFetch({
+            id: grant.id
+          })
+
+          const options: CreateOutgoingPaymentOptions = {
+            walletAddressId: receiverWalletAddress.id,
+            debitAmount,
+            incomingPayment: incomingPayment.toOpenPaymentsTypeWithMethods(
+              receiverWalletAddress
+            ).id,
+            grant
+          }
+
+          for (let i = 0; i < 3; i++) {
+            const payment = await outgoingPaymentService.create(options)
+            assert.ok(!isOutgoingPaymentError(payment))
+            expect(payment.grantSpentDebitAmount.value).toBe(
+              BigInt(debitAmount.value * BigInt(i))
+            )
+            expect(payment.grantSpentReceiveAmount.value).toBe(
+              // Must account for interledger/pay off-by-one issue (even with 0 slippage/fees)
+              BigInt((debitAmount.value - BigInt(1)) * BigInt(i))
+            )
```

Comment:
didn't see an issue for this quirk identified in the comment (not sure where we left it) but figured id note why receive amount should be 1 less than the debit amount here

Reply:
It's a feature, not a bug 😆